src/jsony.nim

```diff
@@ -736,9 +736,11 @@ proc dumpHook*(s: var string, v: string) =
       s.dumpStrFast(v)
 
 template dumpKey(s: var string, v: string) =
-  const v2 = v.toJson() & ":"
```

Comment:
`jsony.toJson(v)` is enough.

Reply:
As you prefer

Reply:
its not a preference, your way adds runtime overhead

Reply:
Euh, the contrary.  The toJson is a proc call, so this is the compiler that will decide or not to inline the call.  after that, the & ":" will cause a need to increase the capacity of the string potentially, so a new alloc and memcopy before appending the ':'.
My proposal immediately allocates enough memory to the key + the ':'+ a bit more in case of escapes.  
the dumpHook is still a proc call that is not guaranteed to be inlined but it is one less call the stack than the toJson (which call it to)
So from the runtime perspective it is less overhead.

Reply:
my bad.. Sorry.  you are right